### DISCUSSION TITLE
i3xrocks scripts configuration

### DISCUSSION THREAD

**Discussion by michaelzusev** [regolith-linux/regolith-desktop#814] [Q&A]
I have copied conf.d from default to config directories as [instructed](https://regolith-desktop.com/docs/howtos/add-remove-blocklets/#customize-bar).

However, I found that what I'd like to configure is in `/usr/share/i3xrocks/scripts/`

I figure I can copy that to my home config as well: `~/.config/regolith2/i3xrocks/scripts`, but when I make changes to the scripts, the status blocks dissapear. I revert the changes, the status blocks do not appear again. I have restarted i3 config, i3, logged off and on, and rebooted; no changes. 

Is there a way to override script files just like conf.d? I want to avoid having to edit them in `/usr/share`

**Comment by comkieffer** [comment]
Take a look at [regolith-i3xrocks-config/conf.d/01_setup](https://github.com/regolith-linux/regolith-i3xrocks-config/blob/87ab5d2c8cab203e8a33fa53f9eb6a965b964b08/conf.d/01_setup). It should be enough to override the `command` key to pass whichever script you want to it.
